shared/webpack/loaders/coffee.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  test: /\.coffee(\.erb)?$/,
```

Comment:
do we really need this as we don't use Coffescript?

Reply:
@olivierobert I kept it because many libraries still use `.coffee` like RailsAdmin